metadata.json

```diff
@@ -106,7 +106,7 @@
   "tags": [
     "ossec",
     "hids",
-    "4.0",
+    "4.2",
```

Comment:
@alberpilot not sure what is this tag used for.

Reply:
Those tags appears when the repository is uploaded to forge:

![image](https://user-images.githubusercontent.com/3830293/135861617-ec94687f-2ea0-4ad3-965a-83d47757fa5f.png)


@vcerenu please change 4.2 accordingly.

Reply:
changed 4.2 to 4.3